### ISSUE TITLE
"Invalid length of the property stream. Parsing .msg file"

### ISSUE THREAD

**Issue by Gergall** [Sicos1977/MsgKit#96]
Hello, thank you for this wonderful programming tool.

I generated some MSG files and tried to upload them into NetDocuments, which is a system that tries to parse uploaded documents so that it can index the different parts of the email for search purposes.

The "Invalid length of the property stream" error message comes from NetDocuments, but it is complaining about the MSG files created by MsgKit.  I was wondering if you have any ideas.

The MSG files open fine and I can even save them to my Inbox in Outlook.  So they seem good to me.  But apparently, behind the scenes, something about these files might be a tiny bit incorrect.  I don't know enough about Outlook MSG files or "property streams" to know what could be wrong.

Thank you.

**Comment by Gergall** [comment]
Thank you for the quick reply.

There might not be anything we can do.  NetDocuments will not provide a more detailed error message so we don't have any clues about what property stream has an issue.

I tried just generating the "Peter Pan" example email using your exact sample code, with no changes (I provided my own crocodile.jpg).  Even that email causes the property stream error.  As usual, the email works fine on my machine and works fine in Outlook, only NetDocuments gives an error.

If by chance you ever find out what the problem might be, I would be interested to know about it.  But I agree that due to the error message being vague, this is a needle in a haystack and probably not solvable right now.
